Formula/p11-kit.rb

```diff
@@ -39,11 +41,10 @@ def install
 
     system "./configure", "--disable-dependency-tracking",
                           "--disable-silent-rules",
-                          "--disable-trust-module",
                           "--prefix=#{prefix}",
                           "--sysconfdir=#{etc}",
                           "--with-module-config=#{etc}/pkcs11/modules",
-                          "--without-libtasn1"
+                          "--with-trust-paths=#{etc}/ca-certificates/cert.pem"
```

Comment:
Out of curiosity... how does this work? Is this Homebrew's own etc installed in your home directory? If so, that seems good. I assume that's what's going on here?

(If you're trying to use the real system /etc here, then that's going to be broken except on your system, since this path is not standard.)

Reply:
`etc` is `$HOMEBREW_PREFIX/etc`.